soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4055,27 +4069,42 @@ void DrawRandoEditor(bool& open) {
 
                     // Cuccos to return
                     SohImGui::EnhancementSliderInt("Cuccos to return: %d", "##RandoCuccosToReturn",
-                                                    "gRandomizeCuccosToReturn", 0, 7, "", 7);
+                                                    "gRandomizeCuccosToReturn", 0, 7, "", 7, true);
                     InsertHelpHoverText(
                         "The amount of cuccos needed to claim the reward from Anju the cucco lady"
                     );
                     ImGui::Separator();
 
                     // Big Poe Target Count
                     SohImGui::EnhancementSliderInt("Big Poe Target Count: %d", "##RandoBigPoeTargetCount",
-                                                    "gRandomizeBigPoeTargetCount", 1, 10, "", 10);
+                                                    "gRandomizeBigPoeTargetCount", 1, 10, "", 10, true);
                     InsertHelpHoverText(
                         "The Poe collector will give a reward for turning in this many Big Poes."
                     );
                     ImGui::Separator();
 
                     // Skip child stealth
-                    SohImGui::EnhancementCheckbox(Settings::SkipChildStealth.GetName().c_str(), "gRandomizeSkipChildStealth");
-                    InsertHelpHoverText(
-                        "The crawlspace into Hyrule Castle goes straight to Zelda, skipping\n"
-                        "the guards."
-                    );
+                    // Disabled when Skip Child Zelda is active
+                    if (disableZeldaRelatedOptions) {
+                        CVar_SetS32("gRandomizeSkipChildStealth", 0);
```

Comment:
Yeah, you're not supposed to ever be able to go back to the courtyard after meeting Zelda. 

This might clear it up a little: https://github.com/HarbourMasters/Shipwright/issues/749

This basically fixes the Skip Child Stealth portion of that issue.

Reply:
Did we confirm that's the case with skip child Zelda in 64rando? It *feels* right but it'd be good to get that behavior documented on the issue

Reply:
N64 rando doesn't have skip child zelda. But just following N64 vanilla logic, you're never able to get back to the courtyard after meeting zelda because the guards after the moat will stop you.

Reply:
> N64 rando doesn't have skip child zelda.

yes it does

![image](https://user-images.githubusercontent.com/70942617/180607704-08cd7473-a173-44cc-ba8a-5387e4e5c55a.png)